### PR TITLE
Drop rsync dependency

### PR DESCRIPTION
**Pull request by sethfitz**

### CODE REVIEW

**Comment by vincentsarago** [comment]
🙆‍♂️

File: Dockerfile [hunk 1]
```diff
@@ -125,7 +124,7 @@ RUN \
   pip3 install -U rasterio --no-binary rasterio -t python/
 
 # delete build deps, symlinks, etc.
-RUN rsync -a lib64/ lib/ && rm -rf lib64/
+RUN cp -aR lib64/ lib/ && rm -rf lib64/
```
Comment (vincentsarago) [inline Dockerfile:127]: @mojodna I think it should be `cp -aR lib64/* lib/` here, sorry If I misguided you

Comment (sethfitz) [reply Dockerfile:127]: Good catch. I was testing on MacOS and it seemed to work as expected; not so on Linux.

Comment (vincentsarago) [reply Dockerfile:127]: > I was testing on MacOS and it seemed to work as expected; not so on Linux.

Same 😄

**Pull request merged**